myscript.js

```diff
@@ -1,29 +1,30 @@
 //list item template
-var itemTemplate = 
+var ITEM_TEMPLATE =
     '<a href="#" class="list-group-item"> \
         <input type="checkbox" onclick="completeItem(this)" value=""> \
         <span class="badge">A</span> \
-        <button class="delete-button" onclick="deleteItem(this)">Delete</button></a>';
+        <button class="delete-button" onclick="deleteItem(this)">Delete</button> \
+        <span class="item-text"></span></a>';
```

Comment:
Just added a `<span class="item-text"></span>` as the place to put the text entered by the user. I'll call out where it's used